lib/plsql/procedure.rb

```diff
@@ -102,7 +102,6 @@ def get_argument_metadata #:nodoc:
       subprogram_id_column = (@schema.connection.database_version <=> [10, 2, 0, 2]) >= 0 ? 'subprogram_id' : 'NULL'
       # defaulted is available just from version 11g
       defaulted_column = (@schema.connection.database_version <=> [11, 0, 0, 0]) >= 0 ? 'defaulted' : 'NULL'
-
```

Comment:
I probably should add some tests for table columns and object attributes of
bfloat and bdouble.